game/Submods/NSFW Submod/nsfw_brb.rpy

```diff
@@ -0,0 +1,106 @@
+# BRB - Going to masturbate
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_brb_nsfw_masturbate",
+            prompt="I'm going to go masturbate",
+            category=['be right back'],
+            conditional="mas_canShowRisque()",
+            pool=True,
+            unlocked=True
+        ),
+        markSeen = True
+    )
+
+label monika_brb_nsfw_masturbate:
+    # make a random int, for randomness
+    $ mas_Rand = renpy.random.randint(1, 3)
+
+    # To do - add in Affectionate variation?
+    # Enamored (400-999+)
+    if mas_isMoniEnamored(higher=True):
```

Comment:
Currently the player can only unlock this topic from 2000 affection and above, due to the conditional above I think. "mas_canShowRisque" does 3 things:
1. It checks if you're above 2000 affection.
2. It checks if you've had your first kiss with Monika.
3. It checks if you are over the age of 18.

Reply:
Ah, I see. I didn't know that canShowRisque did an affection check

Reply:
Yeah, it's pretty nifty. You can also apply your own affection threshold by inputting a new affection cap. So your function would look like: "mas_canShowRisque(aff_thresh=400)".

Reply:
Aha. I might make it so that you can use this interaction any time after starting the mod, so it'd be aff_thresh=0. because I like the idea that someone might install the mod and then immediately try to use this option, because it means monika can have a funny reaction to it.

Reply:
Lol true. If that's the case, you might just be after the mas_isOver18 check rather than the Risque check. Only reason being that it checks for the player's first kiss as well as the other two things, which doesn't unlock till ~400 affection.

Reply:
Lol true. If that's the case, you might be better off using the mas_isOver18 check, since the Risque check includes the first kiss requirement (only unlocks at 400 affection)

Reply:
Does the mas_isOver18 check require a date? it seems to based on the definition

Reply:
You can input a date, which gets used as a comparison (so like, are you 18 by this date inputted). If you input no date, it defaults to None, and assumes that the date you want is today.